openedx/core/djangoapps/user_api/tests/test_helpers.py

```diff
@@ -52,22 +53,35 @@ def test_ignores_expected_errors(self):
 
     @mock.patch('openedx.core.djangoapps.user_api.helpers.LOGGER')
     def test_logs_errors(self, mock_logger):
+        self.maxDiff = None
```

Comment:
Do you want to keep this in the code?  If so, maybe set a highish but not unreasonable value for it.